lib/src/command/moronepo_command.dart

```diff
@@ -12,9 +14,19 @@ abstract class MoronepoCommand<T> extends Command<T> {
 class MoronepoResults {
   final String workingDirectory;
   final String projectName;
+  final List<String> filter;
+  final List<String> dependencies;
 
   MoronepoResults({
     this.workingDirectory,
     this.projectName,
+    this.filter,
+    this.dependencies,
   });
+
+  bool get isDart => filter.contains("isDart") ? true : null;
```

Comment:
`isDart` sounds weird, because flutter project is a dart project too :/ What do you think about `isNonFlutter`? Or maybe its possible to negate filter like `-f isFlutter!` or `-f hasTests!`? Running `print -f hasTests!` can has a practical usage if you want to know how many projects hasn't tests.

Reply:
Done - but I went with `!isFlutter` instead of `isFlutter!` to keep it consistent with negating in code. What do you think?